src/test/groovy/com/athaydes/automaton/SwingBaseForTests.groovy

```diff
@@ -312,6 +314,37 @@ abstract class SwingDriverWithSelectorsTest extends SimpleSwingDriverTest {
 	}
 
 	@Test
+	void testMoveTo_TableCell_CustomRenderer() {
+		def tModel = [
+				[ firstCol: new SimpleStringProperty('1 - 1'), secCol: new SimpleStringProperty('1 - 2') ],
+		]
+		def cellRenderer = new DefaultTableCellRenderer(){
+			@Override
+			protected void setValue(Object value) {
+				setText(((SimpleStringProperty) value).getValue())
+			}
+		}
+		JTable jTable = null
+		new SwingBuilder().edt {
+			jFrame = frame( title: 'Frame', size: [ 300, 200 ] as Dimension,
+					location: defaultLocation, show: true ) {
+				scrollPane {
+					jTable = table {
+						tableModel( list: tModel ) {
+							propertyColumn( header: 'Col 1', propertyName: 'firstCol', type: SimpleStringProperty.class, cellRenderer: cellRenderer )
+							propertyColumn( header: 'Col 2', propertyName: 'secCol', type: SimpleStringProperty.class, cellRenderer: cellRenderer )
+						}
+					}
+				}
+			}
+		}
+		waitForJFrameToShowUp()
+
+		assert withDriver().getAll( 'text:1 - 1' ).size() == 1
+		assert withDriver().getAll( 'text:1 - 2' ).size() == 1
```

Comment:
I didn't remember you just get a FakeComponent back... you're right for the purpose of this feature, this is enough, I guess.